manual/src/refman/extensions/tail_mod_cons.etex

```diff
@@ -0,0 +1,548 @@
+(Introduced in OCaml 4.14)
+
+Note: this feature is considered experimental, and its interface may
+evolve, with user feedback, in the next few releases of the language.
+
+Consider the simple way of implementing the "List.map" function:
```

Comment:
```suggestion
Consider the idiomatic way of implementing the "List.map" function:
```
??

Reply:
meh, "idiomatic" feels jargony to me, so I kept "simple" even if it is less precise.

Reply:
"simple" to me has overtones (in this context, at least) of being naïve, which is unfortunate given that it's the actual implementation in the Standard Library!

Could it just be introduced that way instead? Perhaps `Consider the Standard Library's implementation of "List.map":`? I don't mind if "idiomatic" is too jargony (I don't agree FWIW - it's a quite standard English word), but "simple" doesn't feel appropriate 🤷

Reply:
I replaced "simple" by "natural", is it better?

(Personally I don't care that this implementation is in the standard library, and I would criticize it if made sense. But here the point is that this is the most obvious way to write `List.map`, not that it it is used in this or that library.)

Reply:
"natural" works for me, thanks 🙂

Reply:
> meh, "idiomatic" feels jargony to me

But it is jargon of _our_ field! ;-) For me "idiomatic" means "natural for a given programming language". Of course just "natural" also looks good, but from point of view the decision is too cautious.